### ISSUE TITLE
Building CNI Plugins

### ISSUE THREAD

**Issue by 14rcole** [kubernetes-incubator/cri-o#592]
The cri-o tutorial instructs the user `go get` [containernetworking/cni](https://github.com/containernetworking/cni) and build it from source.  The output is supposed to be

```
Building API
Building reference CLI
Building plugins
   flannel
   tuning
   bridge
   ipvlan
   loopback
   macvlan
   ptp
   dhcp
   host-local
   noop
```
But the output is actually

```
Building API
Building reference CLI
Building plugins
   noop
```

Building works as expected if the user checks out the v0.3 branch before building.  Is this an issue that has to be addressed in CNI or is it something that should be changed in the tutorial?

**Comment by runcom** [comment]
Didn't we fix that https://github.com/kubernetes-incubator/cri-o/pull/589?

**Comment by sameo** [comment]
@runcom we did not fix the tutorial
Fixed with PR #593

**Comment by runcom** [comment]
Alright just saw Sam's PR
